contracts/FantomAuction.sol

```diff
@@ -509,13 +509,8 @@ contract FantomAuction is
                 .mul(platformFee)
                 .div(1000);
 
-            if (auction.payToken == address(0)) {
+            if (auction.payToken != address(0)) {
```

Comment:
The createAuction() still allows address(0) to be used, please update the condition. Also, please remove out-commented function call for placeBid() with native FTM-s. 

I'd rather have the invalid pay tokens sorted out by require() checks on entry functions. If we do that, these conditions should be irrelevant, I believe. We don't have to deal with older auctions, since a new deployment is planed anyway, right? Thanks.

Reply:
Yeah makes sense. When creating auction (removed the allowance of address(0)), we check that the payToken is enabled in the token registry. I think that's good enough and we can remove all the instances like line #512?

Reply:
Please make sure to check the line #234 in createAuction() function. The require() there still allows zero address to be used. Thank you, Ibrahim.